things/api/grpc/endpoint_test.go

```diff
@@ -109,3 +98,18 @@ func TestIdentify(t *testing.T) {
 		assert.Equal(t, tc.code, e.Code(), fmt.Sprintf("%s: expected %s got %s", desc, tc.code, e.Code()))
 	}
 }
+
+func newService(tokens map[string]string) things.Service {
```

Comment:
Please move these private functions at the beginning in order to be consistent (every other test file has private functions at the beginning).

Reply:
I didn't want anything to be before `TestMain` so, in order to be consistent with #283, I've placed private methods at the end of the file.